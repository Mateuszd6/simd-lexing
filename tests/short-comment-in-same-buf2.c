long long long long long long long long long long long long long long long
long long long long long long long long long long long long long long long


// foo
// bar
// scoo
// zoo
// foo
// bar
// scoo \\\\
zoo is sill a comment


long long long long long long long long long long long long long long long
long long long long long long long long long long long long long long long
