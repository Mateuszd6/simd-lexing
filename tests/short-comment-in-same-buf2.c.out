Parsed: 17 lines, 60 ids, 0 strings, 0 chars, 0 ints, 0 hex,   0 floats,    7 //s, 0 /**/s,   0 #foo
